codebuild_cli.py

```diff
@@ -1,129 +1,130 @@
 #!/usr/bin/env python
 import json
 import os
-
-import boto3
 import click
 
-## To do
-## Need some check to see if user has set their AWS creds correctly
-## let user use both https and ssh address for git url
-
-
-def check_success(task, response):
-    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
-        print(f"{task} was a success!")
-    else:
-        print(f"{task} failed!")
-
+from dns_cli import check_response, check_aws_conn
 
-def display_aws_account():
-    client = boto3.client('iam', region_name='eu-west-2')
+AWS_REGION = "eu-west-2"
+DEFAULT_CI_BUILDER = "public.ecr.aws/h0i0h2o7/uktrade/ci-image-builder"
```

Comment:
Should h0i0h2o7 be uktrade?

Reply:
I believe that's the `dev` account. We have now migrated over to using the `tools` account.

```suggestion
DEFAULT_CI_BUILDER = "public.ecr.aws/uktrade/ci-image-builder"
```

Reply:
ah yes, I need to add the new aws tools account